src/program.ts

```diff
@@ -193,6 +195,20 @@ program
         launch(program, options)
     })
 
+program
+    .command('createPlaybook')
```

Comment:
move playbook commands to a name space:

`mamoru-cli playbook init`
`mamoru-cli playbook publish`

etc